### ISSUE TITLE
embedded hyperlinks in Item thumbnail images lack discernible text

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#857]
Embedded hyperlinks in Item thumbnail images on collection-level pages, catalog page, and found-set pages lack discernible text. WCAG guidelines require that links have discernible text.[ Deque help page](https://dequeuniversity.com/rules/axe/4.3/link-name?application=AxeChrome) for this error. 

Page tested: https://ldr.lafayette.edu/collections/lafayette-newspaper-archive 
Associated elements/source code viewed in test: 
```
"<a class=""media-left"" href=""/concern/publications/x346d432j"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/rx913q00j?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/0p096718j"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/m613mx805?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/6m311p42q"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/dz010q19j?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/0k225b29j"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/h989r3467?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/6108vb47n"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/s1784k99j?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/tb09j6054"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/v979v3478?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/gx41mj14n"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/b8515n62b?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/pv63g0481"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/k0698770z?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/dj52w512m"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/jh343s66t?file=thumbnail"">
</a>"
"<a class=""media-left"" href=""/concern/publications/rn3011585"">
        <img class=""hidden-xs file_listing_thumbnail"" src=""/downloads/qj72p747m?file=thumbnail"">
</a>"
```

Other page tested with similar errors resulting: https://ldr.lafayette.edu/catalog 
Associated elements/source code viewed in test: 
```
<a data-context-href="/catalog/7h149r06p/track" href="/concern/publications/7h149r06p"><img src="/downloads/xs55md433?file=thumbnail"></a>
<a data-context-href="/catalog/3b5919997/track" href="/concern/publications/3b5919997"><img src="/downloads/9880vs339?file=thumbnail"></a>
<a data-context-href="/catalog/cn69m5494/track" href="/concern/publications/cn69m5494"><img src="/downloads/c247dt42f?file=thumbnail"></a>
<a data-context-href="/catalog/ft848r88j/track" href="/concern/publications/ft848r88j"><img src="/downloads/5x21tg84f?file=thumbnail"></a>
<a data-context-href="/catalog/p5547s52t/track" href="/concern/publications/p5547s52t"><img src="/downloads/76537264r?file=thumbnail"></a>
<a data-context-href="/catalog/vt150k55d/track" href="/concern/publications/vt150k55d"><img src="/downloads/6d56zx828?file=thumbnail"></a>
<a data-context-href="/catalog/bv73c205w/track" href="/concern/publications/bv73c205w"><img src="/downloads/hh63sx357?file=thumbnail"></a>
<a data-context-href="/catalog/rj430596j/track" href="/concern/publications/rj430596j"><img src="/downloads/r494vm597?file=thumbnail"></a>
<a data-context-href="/catalog/tm70mw211/track" href="/concern/publications/tm70mw211"><img src="/downloads/h702q773v?file=thumbnail"></a>
<a data-context-href="/catalog/j6731520s/track" href="/concern/images/j6731520s"><img src="/downloads/v118rg081?file=thumbnail"></a>

```

**Comment by noraegloff** [comment]
Deque aXe tools severity level = serious

**Comment by rococodogs** [comment]
looks like this is addressed in hyrax 3 by adding an `aria-hidden="true"` attribute to the link and alt text to the thumbnail image

(from github, [comparing hyrax v2.9.6 to v3.4.1](https://github.com/samvera/hyrax/compare/v2.9.6...v3.4.1))

<img width="1674" alt="Screen Shot 2022-04-04 at 9 16 11 AM" src="https://user-images.githubusercontent.com/2744987/161552720-b4b1e1f1-cca7-46c5-ae8a-a1c2a10e1949.png">

going to close this for now as a fixed-upstream-we'll-eventually-get-it
